torch/autograd/function.py

```diff
@@ -319,9 +319,9 @@ class _SingleLevelFunction(
 ):
     @staticmethod
     def forward(ctx: Any, *args: Any, **kwargs: Any) -> Any:
-        r"""
-        This function is to be overridden by all subclasses. There are two ways
-        to define forward:
+        r"""Use this function and override it in all subclasses.
```

Comment:
hmm I'm not sure what this means, I think its okay to not follow strictly the format if we need to reword a passage significantly in this PR

Reply:
(let's revert this change?)